docs/en/connector-v2/source/Rabbitmq.md

```diff
@@ -0,0 +1,148 @@
+# Rabbitmq
+
+> Rabbitmq source connector
+
+## Description
+
+Used to read data from Rabbitmq.
+
+## Key features
+
+- [ ] [batch](../../concept/connector-v2-features.md)
```

Comment:
> Unsupported batch mode?

The consumer of rabitmq source connect obtains the message through the event monitoring and listening method. If there is a message, it will be automatically obtained. it seems that no way to judge whether there is no subsequent event message.

Reply:
thx. agree with you